static/resources/yaml/observability_pipelines/v2/setup/aws_eks.yaml

```diff
@@ -0,0 +1,43 @@
+datadog:
+  site: "datadoghq.com"
+
+## Autoscaling
+##
+autoscaling:
+  enabled: true
+  minReplicas: 2
+  targetCPUUtilizationPercentage: 80
+
+podDisruptionBudget:
+  enabled: true
+  minAvailable: 1
+
+## HorizontalPodAutoscaler (HPA) requires resource requests to function,
+## so this example configures several default values. Datadog recommends
+## that you change the values to match the actual size of instances that
+## you are using.
+resources:
+  requests:
+    cpu: 1000m
+    memory: 512Mi
+
+
+## To prevent a single datacenter from causing a complete system failure,
+## this example spread across your cluster among availability zones.
```

Comment:
I'm not sure I understand this. What is spread across the cluster?

Reply:
> What is spread across the cluster?

Pods are spread across the cluster among the availability zones. Here's some context from the official k8s docs: [Pod Topology Spread Constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/)

I've changed it to:
"To prevent a single datacenter from causing a complete system failure, the `topologySpreadConstraints` in this example controls how OP Worker pods are spread across your cluster among availability zones."

WDYT? Any suggestions how we can re-phrase it are highly appreciated 🙂

Reply:
@krlv your rephrasing is perfect! And thanks for the doc link, it was helpful!